backend/.dockerignore

```diff
@@ -2,10 +2,10 @@
 **/*.sqlite3
 docker-*.yml
 Dockerfile
+Dockerfile.*
 **/*.md
 nginx/
 __pycache__
 ./temp
 *_data.json
-**/test*.py
```

Comment:
Why aren't you ignoring tests? They are not necessary for the application to work, while the weigh in the final image.

Reply:
Yeah, so you're right, they're not necessary for the final image. However, if you'll note for the test step of the CI, rather than running it on the CI machine directly, I'm running it through the docker container with `exec` so that it matches the exact production environment (OS, installed programs, etc). So the tests need to at least be in the container during the build step. We could, in theory, exclude the test files from the image and then in the test step of the CI add them in with docker. But given the negligible size of the test files, I would not prioritize that super high.

Reply:
Why aren't you trusting the portability of python, and we need to test it inside the production container? With compiled languages, it's normal to compile and run tests and then package the release version separately and ship it afterwards (I'm thinking about java and C#).

Reply:
Here inside GitHub Actions, we can test the code on windows, ubuntu and macos. Then, we can fire a docker cp command and get the tests inside the container too if you feel it necessary, The reason of my interest is that I like the idea of having a state-of-the-art docker image, but I don't mind putting the question apart for now.